### DISCUSSION TITLE
General MusicAssistant `media_player` entity

### DISCUSSION THREAD

**Discussion by samuelspagl** [music-assistant/support#1822] [Q&A] [answered]
Hello Music-Assistant team,

your project is very very cool. The integration into HomeAssistant provides me with `media_players` for the different players, which is cool, but I would also be interested in an `media_player` entity of the "main" player.

With "main" player I mean, the virtual player in MusicAssistant that streams to the players.

I would love to see capabilities similar to the [Sonos Lovelace card](https://github.com/johanfrick/custom-sonos-card), which would be a huge advantage for me. 

I would also be open to help developing such a feature, but love to hear your opinions.

Best Samuel

**Comment by OzGav** [comment]
I don’t understand what you mean by main player. MA supports a different queue for every player with those queues playing simultaneously

**Reply by samuelspagl** [reply]
Thanks for the fast response! Ah thats good to know, because I thoughts came from the WebUI. Where you have this one player-bar, which seemed to be a "main" player casting to the other speakers.
<img width="1257" alt="Bildschirmfoto 2024-01-24 um 09 35 28" src="https://github.com/music-assistant/hass-music-assistant/assets/46893256/10f48109-63ae-4faf-b6c3-b9ba4a4eac9b">

But if I misunderstood this, then my question is probably obsolete :)

**Reply by OzGav** [reply, marked as answer]
Yeah so those player controls are currently going to control your Wohnzimmer player. If you press on Wohnzimmer the player list will pop up and you can select a different speaker and then the controls will control that one

**Reply by samuelspagl** [reply]
Yep so I misunderstood it completely. Then my question also doesn't make any sense and this can be closed from my point of view.
